substrate/frame/collective/src/lib.rs

```diff
@@ -676,7 +710,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 	/// Execute immediately when adding a new proposal.
 	pub fn do_propose_execute(
 		proposal: Box<<T as Config<I>>::Proposal>,
-		length_bound: MemberCount,
+		length_bound: u32,
```

Comment:
Why not use `MemberCount` anymore?

Reply:
`length_bound` is a bound for the proposal's encoded length, so that type alias was misplaced, it worked because the underlying type was u32 as well